### DISCUSSION TITLE
Riavvio VM, global leaks non riaprte

### DISCUSSION THREAD

**Discussion by Aeiuoieoie** [globaleaks/globaleaks-whistleblowing-software#3730] [General]
Ciao a tutti, sto facendo dei test su una VM con debian, avevo installato tutto ed era funzionante. Ho riavviato la VM e adesso non trovo più il servizio attivo.

C'è qualcosa da fare per riavviare manualmente o verificare cosa nons tia funzionando? Alla vm ci accedo regolarmente

**Comment by evilaliv3** [comment]
Grazie per la domanda.

Credo semplicemente in seguito all'aggiornamento si possa essere disabilitata l'opzione di autostart del servizio.

Per farlo riparite esegua: service globaleak start

Poi esegua: service globaleaks enable
Con questo comando al prossimo riavvio globaleaks partirà automaticamente
